pack/scw.json

```
@@ -0,0 +1,380 @@
+		"text": "<b>Interrupt:</b> When boost icons on an encounter card are counted, exhaust Scarlet Witch's Crest → increase or decrease the number of boost icons on that card by 1 for this count.",
+		"traits": "Item.",
+		"type_code": "upgrade"
+	},
+	{
+		"attack": 1,
+		"attack_cost": 1,
+		"code": "15010",
+		"cost": 4,
+		"deck_limit": 1,
+		"faction_code": "justice",
+		"flavor": "\"Hey! Geriatrics! Let's get a move on. The Avengers need us!\"",
+		"health": 4,
+		"is_unique": true,
+		"name": "Speed",
+		"pack_code": "scw",
+		"position": 10,
+		"quantity": 1,
+		"resource_physical": 1,
+		"subname": "Thomas Shepherd",
+		"text": "<b>Response</b>: After Speed thwarts, ready him. (Limit once per round.)",
+		"thwart": 2,
+		"thwart_cost": 1,
+		"traits": "Avenger.",
+		"type_code": "ally"
+	},
+	{
+		"attack": 1,
+		"attack_cost": 1,
+		"code": "15011",
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "justice",
+		"flavor": "\"I'm the child of a witch and an android. I'm their wish come true.\"",
+		"health": 3,
+		"illustrator": "Steve Ellis",
+		"is_unique": true,
+		"name": "Wiccan",
+		"pack_code": "scw",
+		"position": 11,
+		"quantity": 1,
+		"resource_mental": 1,
+		"subname": "William Kaplan",
+		"text": "<b>Response:</b> After Wiccan thwarts, discard the top card of the encounter deck. For each boost icon discarded this way, deal 1 damage to an enemy.",
+		"thwart": 1,
+		"thwart_cost": 1,
+		"traits": "Avenger.",
+		"type_code": "ally"
+	},
+	{
+		"code": "15012",
+		"cost": 3,
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"name": "Crisis Averted",
+		"pack_code": "scw",
+		"position": 12,
+		"quantity": 3,
+		"resource_mental": 1,
+		"text": "<b>Hero Action</b> <i>(thwart)</i>: Remove 6 threat from the main scheme. If you paid for this card using a [mental] resource, this thwart ignores the crisis icon ([crisis]).",
+		"traits": "Thwart.",
+		"type_code": "event"
+	},
+	{
+		"code": "15013",
+		"cost": 1,
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"flavor": "\"A hero isn't the one who always wins. They're the one who always tries.\" - Spider-Man",
+		"illustrator": "Andrea Di Vitto & Laura Villari",
+		"name": "Multitasking",
+		"pack_code": "scw",
+		"position": 13,
+		"quantity": 3,
+		"resource_physical": 1,
+		"text": "<b>Hero Action</b> <i>(thwart)</i>: Remove 2 threat from a scheme. If you paid for this card using a [mental] resource, remove 2 threat from a different scheme.",
+		"traits": "Thwart.",
+		"type_code": "event"
+	},
+	{
+		"code": "15014",
+		"cost": 1,
+		"deck_limit": 3,
+		"faction_code": "justice",
+		"flavor": "\"Return what you've taken and I'll let you go. This time. But next time, you won't be so lucky. And I won't be so kind.\" - Moon Knight",
+		"name": "Swift Retribution",
+		"pack_code": "scw",
+		"position": 14,
+		"quantity": 3,
+		"resource_mental": 1,
+		"text": "<b>Hero Action</b> <i>(attack)</i>: The villain schemes. Deal 4 damage to the villain.",
+		"traits": "Thwart.",
```

This trait should be "Attack."

```suggestion
		"traits": "Attack.",
```